bots/happinessncheer.bot.js

```diff
@@ -0,0 +1,48 @@
+const Slack = require('pico-slack');
+const cron = require('node-schedule');
+const {differenceInCalendarDays} = require('date-fns');
+
+const peeps = [
+	`Rbb`,
```

Comment:
Shouldn't we use actual slack usernames here so we can @-mention people?

Reply:
Actually @'ing people via `pico-slack` is weird? It use to work, then it didn't (you have to use their user id in a hyperlink markup format), I'm honestly not sure where it is now because Slack changes it's sit all the time. 

Since this is a very low impact name-call, I think sticking with pure strings of their names is sufficient.

Reply:
Yeah I've had to do the username-to-id mapping thing, if you want an example I know I've done it in the nation states bot